packages/token-providers/src/fromSso.ts

```diff
@@ -71,7 +71,7 @@ export const fromSso =
       ssoToken = await getSSOTokenFromFile(ssoSessionName);
     } catch (e) {
       throw new TokenProviderError(
-        `The SSO session associated with this profile is invalid. ${REFRESH_MESSAGE}`,
+        `The SSO session associated with this profile is invalid. ${REFRESH_MESSAGE}\n${e}`,
```

Comment:
The refresh message provides relevant information. I don't think we should rethrow error here.
```suggestion
        `The SSO session associated with this profile is invalid. ${REFRESH_MESSAGE}`,
```

Reply:
I ended up removing the underlying error, but changed the wording of the thrown error to hint at the file not found issue.